src/tests/roc_core/test_heap_arena.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2023 Roc Streaming authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <CppUTest/TestHarness.h>
+
+#include "roc_core/heap_arena.h"
+#include "roc_core/memory_ops.h"
+
+namespace roc {
+namespace core {
+
+TEST_GROUP(heap_arena) {
+    void setup() { core::HeapArena::set_flags(core::DefaultHeapArenaFlags
+                                              & ~core::HeapArenaFlag_EnableGuards);
+}
+void teardown() {
+    core::HeapArena::set_flags(core::DefaultHeapArenaFlags
+                               | core::HeapArenaFlag_EnableGuards);
+}
```

Comment:
Seems clang-format broke formatting.

Please use

```
// clang-format off
// clang-format on
```

as we do it in `src/tests/roc_core/test_printer.cpp`